lib/mpl_toolkits/tests/test_mplot3d.py

```diff
@@ -1643,6 +1659,83 @@ def test_computed_zorder():
         ax.axis('off')
 
 
+def test_format_coord():
+    fig = plt.figure()
+    ax = fig.add_subplot(projection='3d')
+    x = np.arange(10)
+    ax.plot(x, np.sin(x))
+    fig.canvas.draw()
+    assert ax.format_coord(0, 0) == 'x=1.8066, y=1.0367, z=−0.0553'
+    # Modify parameters
+    ax.view_init(roll=30, vertical_axis="y")
+    fig.canvas.draw()
+    assert ax.format_coord(0, 0) == 'x=9.1651, y=−0.9215, z=−0.0359'
+    # Reset parameters
+    ax.view_init()
+    fig.canvas.draw()
+    assert ax.format_coord(0, 0) == 'x=1.8066, y=1.0367, z=−0.0553'
+
+
+def test_get_axis_position():
+    fig = plt.figure()
+    ax = fig.add_subplot(projection='3d')
+    x = np.arange(10)
+    ax.plot(x, np.sin(x))
+    fig.canvas.draw()
+    assert ax.get_axis_position() == (False, True, False)
```

Comment:
If you understand what this does, it could do with a docstring. It appears unused though.

Reply:
I cannot really say that I do, no...

Reply:
Maybe it should be deprecated? This was deprecated as a renderer property in 3.4 and removed now: https://github.com/matplotlib/matplotlib/blob/main/doc/api/next_api_changes/removals/00001-DS.rst